src/Mod/Fem/Gui/AppFemGui.cpp

```diff
@@ -25,35 +25,26 @@
 #include <Base/Console.h>
 #include <Base/PyObjectBase.h>
 #include <Gui/Application.h>
-#include <Gui/WidgetFactory.h>
```

Comment:
I don't know who or what has modified this file but as long as Fem is not listed in the .pre-commit-config.yaml file any formatting changes should be avoided.

Reply:
This happened when saving the file in VSCode, which is configured to apply the clang rules on saving. The modification was the removal of the material preferences panel, in favour of moving it to a Materials preferences page.